README.md

```diff
@@ -69,9 +69,9 @@ Winit provides the following features, which can be enabled in your `Cargo.toml`
 
 ## MSRV Policy
 
-The Minimum Supported Rust Version (MSRV) of this crate is **1.65**.
+The Minimum Supported Rust Version (MSRV) of this crate is **1.65**. Changes to the MSRV will be released as patch version updates to `winit`. Once `winit` v1.0 is released, they will instead be released as minor version updates.
```

Comment:
Do we really want to only require patch versions to update `winit`? E.g. changing MSRV between v0.28.6 and v0.28.7? I would be in favor of requiring a minor version bump, e.g. v0.28.6 -> v0.29.

Reply:
A major version bump should only happen for semver compatibility breaking changes, which an MSRV bump does not count as. Before 1.0, the "patch version" is effectively acting as the minor version.

Reply:
changing msrv is not considered a breaking change actually if it's within the policy of the project and behind by some amount.

cargo just has very poor support for the MSRV stuff and it can't account for it based on the rust version client is using. Like all of that is not an issue, because binary clients have `Cargo.lock` which prevents any sort of breakages like that.

Reply:
> A major version bump should only happen for semver compatibility breaking changes, which an MSRV bump does not count as. Before 1.0, the "patch version" is effectively acting as the minor version.

I don't disagree that before v1.0 minor versions are used for semver breaking changes, so if we use a minor version after and before v1.0 to break MSRV it's not the same. But the reason why many libraries handle it that way is to provide compatibility without preventing patchability.

E.g. if you use the minor version after and before v1.0 to break MSRV, consumers who care about MSRV can depend on the library but still get patches. `winit = "0.28.6"` can still be patched with v0.28.7. `winit = "~1.2.2"` can still be patched with v1.2.3.

But if you use patch versions before v1.0 to break MSRV, consumers who care about MSRV have to do this:
`winit = "=0.28.6"`, now we can't release patches of Winit with that MSRV.

Also I'm not sure why we would want to break MSRV in patch versions before 1.0 considering the tight release cycle, which is probably gonna be different after v1.0.

Reply:
Respond to @kchibisov from https://github.com/rust-windowing/winit/pull/3046#discussion_r1306673149:
I also think we should at least require a minor version bump to break MSRV, which is exactly what we should write in that paragraph.

Reply:
Just say that `patch` bumps can't have `msrv` bump, that's all how it should sound like, everything else can have an msrv bump.

Reply:
```suggestion
The Minimum Supported Rust Version (MSRV) of this crate is **1.65**. A change will be accompanied by at least a minor version bump.
```

Reply:
Like that it's also fine, but then you justify `0.28.6 -> 0.28.7` bump, but we don't bump like that and never will.

Reply:
> Like that it's also fine, but then you justify `0.28.6 -> 0.28.7` bump, but we don't bump like that and never will.

How does that justify that? It says "minor", v0.28.6 -> v0.28.7 is a "patch" version bump.

Reply:
Before `1.0` it's basically a minor, so you can add API in them.

Reply:
I understand what you mean, but the terminology still stays the same before and after v1.0.
AFAIK https://semver.org doesn't specify that before v1.0 `PATCH` is labeled `MINOR`.

I mean it's very clear to me, but maybe this is confusing for others? Feel free to suggest something else.

Reply:
I guess, maybe it's fine like that. I've suggest that  `msrv can't be bumped on patch version bumps`